plugin.py

```diff
@@ -147,6 +147,14 @@ def log_solar_charger(
                             "path": f"electrical.solar.{id_}.chargingMode",
                             "value": data.get_charge_state().name.lower(),
                         },
+                        {
+                            "path": f"electrical.solar.{id_}.panelPower",
+                            "value": data.get_solar_power(),
+                        },
+                        {
+                            "path": f"electrical.solar.{id_}.loadCurrent",
+                            "value": data.get_external_device_load(),
```

Comment:
Does this do the right thing if `data.get_external_device_load()` => `None`? Should we just skip the value in that case?

Reply:
Looking at https://github.com/keshavdv/victron-ble/blob/main/victron_ble/devices/solar_charger.py#L78 there is no if-statement in there to catch the "N/A value" of 0x1FF and return null or 0 like there is in https://github.com/keshavdv/victron-ble/blob/main/victron_ble/devices/dcdc_converter.py#L71-L72

So either we do that on this side of the code, or file a PR against Keshav's library.

IRL, my SmartSolar MPPT 75/15 is currently sending a value of "0" for the external load current, and I know for a fact there's nothing connected to the load output.

Reply:
just to be on the safe side: https://github.com/keshavdv/victron-ble/pull/25 - "catch N/A value for solar charger load output"